pkg/crc/api/api_client_test.go

```diff
@@ -20,7 +20,7 @@ import (
 
 type testClient struct {
 	*apiClient.Client
-	config     crcConfig.Storage
+	config     *crcConfig.Config
```

Comment:
since this is just for the unit test, i change the type, but might be better to compose a new interface that embeds the `crcConfig.Storage` and adds `AllSettings()` method

Reply:
I would add it to `crcConfig.Schema` rather than `crcConfig.Storage`, which probably means adding a `schema crcConfig.Schema` field, which would be the same value as `config` (hopefully that's not too odd? :-/ )

Reply:
@cfergeau updated as i mentioned in the first comment, we only need it for the test, so do not want to change the interface in the main code, PTALA